src/components/Input.tsx

```diff
@@ -21,6 +21,7 @@ export const InputField = styled.label`
   display: flex;
   gap: 0.5em;
   flex-direction: column;
+  font-weight: 700;
```

Comment:
same comment here

Reply:
This is how I was trying to make the labels bold for the Name and Description fields when you Add or Edit a Plan. I didn't see any regular weight labels which would be messed up by this update.

Reply:
hmm I think I already added the styling for this in my PR. They're not `font-weight: 700` tho, just `font-weight: 500`
<img width="596" alt="Screen Shot 2022-08-24 at 1 31 46 PM" src="https://user-images.githubusercontent.com/40800353/186486216-96777695-b476-4ec5-92db-611bf5b298fa.png">